documentation/examples/remote_storage/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/prometheus/prometheus/documentation/examples/remote_storage
 
-go 1.20
+go 1.21
```

Comment:
We normally want to keep compatibility with the latest two Go versions. Since 1.22 is not out, we should not update this quite yet.